client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -2,38 +2,51 @@
  * External dependencies
  */
 import { __ as NO__ } from '@wordpress/i18n';
-import { TextControl, SelectControl } from '@wordpress/components';
+import { SelectControl } from '@wordpress/components';
 import React from 'react';
 
 /**
  * Internal dependencies
  */
 import { SiteType } from '../store/types';
 import { useOnboardingDispatch, useOnboardingState } from '../store';
+import './style.scss';
 
 export default function OnboardingEdit() {
 	const { siteTitle, siteType } = useOnboardingState();
 	const { setSiteType, setSiteTitle } = useOnboardingDispatch();
 
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	return (
-		<>
-			<p>{ NO__( "Let's set up your website -- it takes only a moment" ) }</p>
-			{ NO__( 'I want to create a website ' ) }
-			<SelectControl< SiteType >
-				onChange={ setSiteType }
-				options={ [
-					{ label: NO__( 'with a blog.' ), value: SiteType.BLOG },
-					{ label: NO__( 'for a store.' ), value: SiteType.STORE },
-					{ label: NO__( 'to write a story.' ), value: SiteType.STORY },
-				] }
-				value={ siteType }
-			/>
+		<div className="onboarding__questions">
+			<h2 className="onboarding__questions-heading">
+				{ NO__( "Let's set up your website – it takes only a moment." ) }
+			</h2>
+
+			<label className="onboarding__question">
+				<span>{ NO__( 'I want to create a website ' ) }</span>
+				<SelectControl< SiteType >
+					onChange={ setSiteType }
+					options={ [
+						{ label: NO__( 'with a blog.' ), value: SiteType.BLOG },
+						{ label: NO__( 'for a store.' ), value: SiteType.STORE },
+						{ label: NO__( 'to write a story.' ), value: SiteType.STORY },
+					] }
+					value={ siteType }
+					className="onboarding__question-input"
+				/>
+			</label>
 			{ ( siteType || siteTitle ) && (
-				<>
-					<p>{ NO__( "It's called" ) }</p>
-					<TextControl onChange={ setSiteTitle } value={ siteTitle } />
-				</>
+				<label className="onboarding__question">
+					<span>{ NO__( "It's called" ) }</span>
+					<input
+						className="onboarding__question-input"
+						onChange={ setSiteTitle }
```

Comment:
Gutenberg controls process the event to give you that value. We'll need to change this handler.

I'll address that.